docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     environment:
       # Prevents pipenv from loading env files (python dotenv and docker-compose compatibility issue).
       - PIPENV_DONT_LOAD_ENV=1
-    command: echo "I'M A WAGTAIL APP!"
+    command: pipenv run python manage.py runserver 0.0.0.0:8000
```

Comment:
OK. The current `env.default` works just fine for me though - perhaps because it doesn't have any double quotes in it? I've changed it to look for `.docker.env` anyway.